plugin/health_check_test.go

```diff
@@ -20,7 +20,7 @@ func TestPluginHealthCheck(t *testing.T) {
 	for name, f := range map[string]func(*testing.T){
 		"PluginHealthCheck_Success":                 testPluginHealthCheck_Success,
 		"PluginHealthCheck_PluginPanicProcessCheck": testPluginHealthCheck_PluginPanicProcessCheck,
-		"PluginHealthCheck_RPCPingFail":             testPluginHealthCheck_RPCPingFail,
+		// "PluginHealthCheck_RPCPingFail":             testPluginHealthCheck_RPCPingFail,
```

Comment:
Maybe add alink to https://mattermost.atlassian.net/browse/MM-15706? Then it's clear what the process of re-enabling this test is.

Reply:
I am not sure what you mean, there's a PR link in the Jira ticket